packages/terra-notification-dialog/src/NotificationDialog.jsx

```diff
@@ -156,16 +156,15 @@ const NotificationDialog = (props) => {
   /* eslint-disable jsx-a11y/no-noninteractive-tabindex */
   return (
     <AbstractModal
-      ariaLabel={signalWord}
-      aria-labelledby="notification-dialog-signal-word"
-      aria-describedby={dialogTitle ? 'notification-dialog-title' : 'notification-dialog-signal-word'}
-      role="alertdialog"
+      ariaLabelledBy="notification-dialog-signal-word"
+      role={signalWord === 'Alert' ? 'alertdialog' : 'dialog'}
```

Comment:
this should not be dependent on signalWord instead we should use `variant` dialog needs to added only for custom variant

Reply:
We will keep that for the other a11y changes. Reverted this to alertdialog